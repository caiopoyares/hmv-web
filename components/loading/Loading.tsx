import { Box, Spinner } from "@chakra-ui/react";

export const Loading = () => (
  <Box
    position="absolute"
    height="100vh"
    left="50%"
    transform="translate(-50%, 40%)"
  >
    <Spinner
      thickness="4px"
      speed="0.65s"
      emptyColor="gray.200"
      color="blue.500"
      size="xl"
    />
  </Box>
);
