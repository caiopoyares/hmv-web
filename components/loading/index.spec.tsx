import { render } from "@testing-library/react";
import { Loading } from "./Loading";

describe("Loading", () => {
  it("should render default loading spinner", () => {
    const { container } = render(<Loading />);
    expect(container.firstChild).toMatchSnapshot();
  });
});

export {};
