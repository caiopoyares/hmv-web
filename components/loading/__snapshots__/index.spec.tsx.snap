// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render default loading spinner 1`] = `
<div
  class="css-qh99lo"
>
  <div
    class="chakra-spinner css-u4uvb9"
  >
    <span
      class="css-f8n5zr"
    >
      Loading...
    </span>
  </div>
</div>
`;
